src/utils/helpers.js

```diff
@@ -23,6 +23,19 @@ const utils = {
     return cleanedParams;
   },
 
+  toSnakeCase: (camelCasedStr) => camelCasedStr.replace(/[A-Z]/g, (prefix) => `_${prefix.toLowerCase()}`),
+
+  toSnakeCaseKeys: (camelCasedObj, toRecurse = false) => {
+    const snakeCasedObj = {};
+    Object.keys(camelCasedObj).forEach((key) => {
+      const newKey = utils.toSnakeCase(key);
+      snakeCasedObj[newKey] = toRecurse && typeof camelCasedObj[key] === 'object'
```

Comment:
Would arrays nested under an object cause issues here? Since they are also `object`s?
```
const a = {
  somethingSomething: 'a',
  somethingElse: ['a', 'v'],
  somethingNot: { a: 'b' },
};
```

Reply:
Nice catch, yea it will convert arrays into object. Should be simple to just check if its array then not do.